features/rest_register.feature

```diff
@@ -0,0 +1,24 @@
+Feature: As a restaurant owner
+  In order to display my restaurant in the system
+  I would like to be able to create a restaurant account
+
+  Background:
+    Given the following restaurant categories exist
+      | name      |
+      | Sushi     |
+      | Fast food |
+      | Mexican   |
+
+  Scenario:
+    Given I am logged in with "o.dania@aol.com", password '12345678' and password_confirmation '12345678'
+    Then I should see "Are you a restaurant owner? Click to register your restaurant" link
+    And I click on the "Are you a restaurant owner? Click to register your restaurant" link
+    Then I should see "Restaurant Registration Form"
```

Comment:
I would suggest minimizing your assertions. Again, if the form is not on the page the following step would fail. This assertion feels redundant.